Sources/SwiftFusion/Geometry/Rot2.swift

```diff
@@ -48,23 +48,25 @@ extension Rot2: CustomDebugStringConvertible {
   }
 }
 
+/// Group actions
```

Comment:
The standard style is to always end doc comments with a period, even when it's not a complete sentence (https://swift.org/documentation/api-design-guidelines/ > Write a documentation comment "More details" > Begin with a summary "More details")

Reply:
Done.